lib/rpcserver/Service.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+class Service {
```

Comment:
Maybe call this class something like "Methods"? That seems a bit more descriptive.

Reply:
Service is the usual name for this kind of layer. Methods sounds even more generic to me, but RpcMethods might work.

Reply:
I like RpcMethods.